templates/types/streaming/fastapi/app/workflows/__init__.py

```
@@ -0,0 +1 @@
+from .agent import create_workflow
```

Ruff (0.8.2)

`.agent.create_workflow` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)